### ISSUE TITLE
Global uninstalls local version if both (local and global installs) exist

### ISSUE THREAD

**Issue by pawelkuras** [lukesampson/scoop#2608]
calling
```
scoop uninstall <app> -g
```
uninstalls local app 1st.  Should uninstall global.

**Comment by Ash258** [comment]
Git??

**Comment by pawelkuras** [comment]
> calling
> 
> ```
> scoop uninstall <app> -g
> ```
> uninstalls local app 1st. Should uninstall global.

**Comment by pawelkuras** [comment]
:) sorry, updated.

**Comment by Ash258** [comment]
Could reproduced: 

```
system32 $ scoop install aria2 -g
Installing 'aria2' (1.34.0-1) [64bit]
aria2-1.34.0-win-64bit-build1.zip (2.0 MB) [==================================================================] 100%
Checking hash of aria2-1.34.0-win-64bit-build1.zip ... ok.
Extracting aria2-1.34.0-win-64bit-build1.zip ... done.
Linking C:\Scoop\apps\aria2\current => C:\Scoop\apps\aria2\1.34.0-1
Creating shim for 'aria2c'.
'aria2' (1.34.0-1) was installed successfully!
system32 $ scoop list
Installed apps:

  7zip 18.05
  aria2 1.34.0-1 *global*
  aria2 1.34.0-1
  ...

system32 $ scoop uninstall aria2 -g
Uninstalling 'aria2' (1.34.0-1).
Removing shim for 'aria2c'.
Unlinking D:\SCOOP\apps\aria2\current
'aria2' was uninstalled.
system32 $ scoop list
Installed apps:

  7zip 18.05
  aria2 1.34.0-1 *global*
  ...

system32 $ scoop uninstall aria2 -g
Uninstalling 'aria2' (1.34.0-1).
Removing shim for 'aria2c'.
Unlinking C:\Scoop\apps\aria2\current
'aria2' was uninstalled.
system32 $ scoop list
Installed apps:

  7zip 18.05
  ...
```

**Comment by r15ch13** [comment]
That's a bug, I'm one responsible for it https://github.com/lukesampson/scoop/commit/84bd16b82d983c42d7cdc7365d887f55dcfe5cac :grin:

**Comment by JanPokorny** [comment]
Can confirm, reproducable on latest version.

![obrazek](https://user-images.githubusercontent.com/4580066/48772531-f603f600-ecc4-11e8-82ce-c57fc5c30aba.png)

**Comment by Ash258** [comment]
@r15ch13 Could you look into it please? It's pretty annoying 😃

**Comment by rasa** [comment]
Fixed in e51343076613cad9aaa6a0177804958d4c28e00f
